p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/RestoreWalletViewModel.swift

```diff
@@ -31,7 +31,8 @@ final class RestoreWalletViewModel: BaseViewModel {
 
     init(provider: OnboardingStateMachineProvider = OnboardingStateMachineProviderImpl()) {
         let accountStorage: AccountStorageType = Resolver.resolve()
-        deviceShare = available(.mockedDeviceShare) ? "someDeviceShare" : accountStorage.deviceShare
+        deviceShare = available(.mockedDeviceShare) ? OnboardingConfig.shared.mockDeviceShare : accountStorage
```

Comment:
Move `.mockedDeviceShare` to `OnboardingConfig.swift` please. We don't use feature toggle for mocking anymore anymore.

Reply:
kk